pages/common/p5.md

```diff
@@ -0,0 +1,19 @@
+# p5-manager
```

Comment:
Shouldn't this be `p5` ? Since the command name is p5.

Reply:
Yeah wasn't sure what was correct to use here - the actual package is called p5-manager (as p5 is a seperate javascript library). Should this be changed to the command name or package name?

Reply:
I think the idea is that it's the command name - which should be the same as the filename IIRC.

Reply:
Ok, most recent commit has name changed 😁